src/bin/main.rs

```diff
@@ -50,12 +57,57 @@ fn core() -> Result<i32, String> {
                     _ => None,
                 },
             )?;
+
+            println!("    Done!");
```

Comment:
```suggestion
            println!("A new Bonnie configuration file has been initialized at [PATH]!");
```

Please tell the user where their new configuration file is, even though it's a little redundant.